.github/workflows/c-zephyr-tests.yml

```diff
@@ -58,4 +58,7 @@ jobs:
           ref: ${{ inputs.runtime-ref }}
         if: ${{ inputs.runtime-ref }}
       - name: Perform Zephyr tests for C target with default scheduler
-        run: ./gradlew test --tests org.lflang.tests.runtime.CZephyrTest.runZephyrTests
+        run: |
+          ./gradlew test --tests org.lflang.tests.runtime.CZephyrTest.buildZephyrTests
+          ./gradlew test --tests org.lflang.tests.runtime.CZephyrTest.buildGenericTests
```

Comment:
Could we just a single line `./gradlew test --tests org.lflang.tests.runtime.CZephyrTest.*` instead of two lines? I recall that wildcards are accepted.